Updates log categorization docs (#1042)
elastic/observability-docs: docs/en/observability/categorize-logs.asciidoc

millions of log lines into just a few categories.

Within the {logs-app}, the *Categories* page enables you to identify patterns in
your log events quickly. Instead of manually identifying similar logs, the logs 
categorization view lists log events that have been grouped based on their 
messages and formats so that you can take action quicker.

[[create-log-categories]]
== Create log categories

> Create a 
> {ml-docs}/ml-configuring-categories.html[{ml} job to categorize log messages] 
> automatically. {ml-cap} observes the static parts of the message, cluster 
> similar messages, classify them into message categories, and detect unusually 
> high message counts in the categories. 

Reviewer: ```suggestion
Create a 
{ml-docs}/ml-configuring-categories.html[{ml} job to categorize log messages] 
automatically. {ml-cap} observes the static parts of the message, clusters 
similar messages, classifies them into message categories, and detects unusually 
high message counts in the categories. 
```

[role="screenshot"]
image::images/log-create-categorization-job.jpg[Configure log categorization job]

> 1. Select *Categories*, and you are prompted to use {ml}, which carries out the 
>    log rate categorizations.

Reviewer: ```suggestion
1. Select *Categories*, and you are prompted to use {ml} to create 
   log rate categorizations.
```

> 2. Choose a time range for the {ml} analysis. By default, the {ml} job analizes 
>    log messages no lder than four weeks and continues indefinitely.

Reviewer: ```suggestion
2. Choose a time range for the {ml} analysis. By default, the {ml} job analyzes 
   log messages no older than four weeks and continues indefinitely.
```

> 3. Add the indices that contain the logs you want to analyze.

Reviewer: ```suggestion
3. Add the indices that contain the logs you want to examine.
```

> 4. Click *Create ML job*. The job is created and it starts run. It takes a few 

Reviewer: ```suggestion
4. Click *Create ML job*. The job is created, and it starts to run. It takes a few 
```

   minutes for the {ml} robots to collect the necessary data. After the job 
   processed the data, you can view the results.

[[analyze-log-categories]]
== Analyze log categories

The *Categories* page lists all the log categories from the selected indices. 
You can filter the categories by indices. The screenshot below shows the 
categories from the `elastic.agent` log.

[role="screenshot"]

> image::images/log-categories.jpg[Log categories]

Reviewer: Should this not be `.png`?

Author: I pushed a log-categories.jpg in this PR along with the textual changes, so the extension of this particular image is correct here.

> The category row contains the following information: 

Reviewer: You need an extra line below for the bullet points to render properly.

Author: ```suggestion
The category row contains the following information: 

```

* message count: shows how many messages belong to the given category.

> * trend: indicates how the occurrance of the messages changes in time.

Reviewer: ```suggestion
* trend: indicates how the occurrence of the messages changes in time.
```

* category name: it is the name of the category and is derived from the message 
  text.
* datasets: the name of the datasets where the categories are present.
* maximum anomaly score: the highest anomaly score in the category.

> You can view the individual log messages under a particular category by clicking 
> the arrow at the end of the row. To further examine an individual message, it 
> can be viewed in the corresponding log event on the *Stream* page or can be 
> displayed in its context.

Reviewer: ```suggestion
To view al log message under a particular category, click  
the arrow at the end of the row. To further examine a message, it 
can be viewed in the corresponding log event on the *Stream* page or displayed in its context.
```
